src/__tests__/__snapshots__/App.spec.jsx.snap

```diff
@@ -916,13 +978,34 @@ exports[`initial state renders without crashing 1`] = `
                     <input
                       className="a-text-input"
                       id="company_received-through"
-                      max={2020-05-05T04:00:00.000Z}
-                      min={2011-12-01T12:00:00.000Z}
+                      max="2020-05-05"
+                      min="2011-12-01"
+                      onBlur={[Function]}
                       onChange={[Function]}
+                      onKeyDown={[Function]}
                       placeholder="M/D/YYYY"
-                      size="15"
+                      type="date"
                       value=""
                     />
+                    <button
+                      className="a-btn a-btn__link"
+                      onClick={[Function]}
+                    >
+                      <svg
+                        className="cf-icon-svg"
+                        viewBox="0 0 718.9 1200"
+                        xmlns="http://www.w3.org/2000/svg"
+                      >
+                        <path
+                          d="M451.4 613.7l248.1-248.1c25.6-25.1 26-66.3.8-91.9s-66.3-26-91.9-.8l-.8.8-248.1 248.1-248.1-248.1c-25.4-25.4-66.5-25.4-91.9 0s-25.4 66.5 0 91.9l248.1 248.1L19.5 861.8c-25.6 25.1-26 66.3-.8 91.9s66.3 26 91.9.8l.8-.8 248.1-248.1 248.1 248.1c25.4 25.4 66.5 25.4 91.9 0s25.4-66.5 0-91.9L451.4 613.7z"
+                        />
+                      </svg>
+                      <span
+                        className="u-visually-hidden"
+                      >
+                        Clear
```

Comment:
This is visually hidden, so probably would be better to be more verbose and say to "Clear" what.

Maybe this?

"Clear company received through date"